packages/re2.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Re2 < Package
+  description 'RE2 is a fast, safe, thread-friendly alternative to backtracking regular expression engines 
+like those used in PCRE, Perl, and Python.'
```

Comment:
Can we make this one line?  I think the wrapping may cause a problem.

Reply:
Removed